docs/odh/llama-stack-with-odh.md

```diff
@@ -0,0 +1,170 @@
+# Running LlamaStack Operator with ODH
+
+This guide provides instructions for deploying and using LlamaStack with ODH/OpenShift AI Kserve component.
+
+## Prerequisites
+
+- ODH/OpenShift AI [installed](https://github.com/opendatahub-io/opendatahub-operator?tab=readme-ov-file#installation).
+- LlamaStack Operator [installed](../../README.md#installation)
+- Cluster configured to use GPUs (g4dn.xlarge GPU nodes recommended)
+
+## Deploying vLLM via KServe
+
+### 1. Setup KServe in Standard Mode
+
+#### Configure DSCI
+In the DSCI resource, set the serviceMesh managementState to Removed:
+
+```yaml
+spec:
+  applicationsNamespace: redhat-ods-applications
+  monitoring: opendatahub
+    managementState: Managed
+    namespace: opendatahub
+  serviceMesh:
+    controlPlane:
+      metricsCollection: Istio
+      name: data-science-smcp
+      namespace: istio-system
+    managementState: Removed
+```
+
+#### Configure DSC
+In the DSC resource, configure the kserve component:
+
+```yaml
+spec:
+  components:
+    kserve:
+      defaultDeploymentMode: RawDeployment
+      RawDeploymentServiceConfig: Headed
+      managementState: Managed
+      serving:
+        managementState: Removed
+        name: knative-serving
+```
+
+Verify the setup by checking that `kserve-controller-manager` and `odh-model-controller` pods are running:
+
+```bash
+oc get pods -n opendatahub | grep -E 'kserve-controller-manager|odh-model-controller'
+```
+
+### 2. Deploy LLaMA 3.2 Model via KServe UI
+
+1. Create a Connection:
+   - Go to ODH dashboard -> Create Project -> Connections tab -> Create connection
+   - **Project Name:** llamastack
+   - **Connection Type:** URI - v1
+   - **Connection name:** llama-3.2-3b-instruct
+   - **URI:** `oci://quay.io/redhat-ai-services/modelcar-catalog:llama-3.2-3b-instruct` (5.9 GiB)
```

Comment:
Is the 5.9 GiB important here ? Asking because the reader my wonder whether they should reserve some extra space for that or so. Also, I thought it's just about a reference to a model, not downloading it ?

Reply:
Sorry for my ignorance, but what is this connection needed ? Needs it to be the same like the model deployment name below ?